infra/cifuzz/actions/build_fuzzers/build_fuzzers_entrypoint.py

```diff
@@ -40,7 +41,6 @@ def main():
     OSS_FUZZ_PROJECT_NAME: The name of OSS-Fuzz project.
     GITHUB_REPOSITORY: The name of the Github repo that called this script.
     GITHUB_SHA: The commit SHA that triggered this script.
-    GITHUB_REF: The pull request reference that triggered this script.
```

Comment:
Actually, could you describe `GITHUB_EVENT_PATH` here?

Reply:
Will do

Reply:
Done